### PR TITLE
[DOC-1235] GameDay documentation

### PR DESCRIPTION
**Pull request by SmritiSatya**
Add GameDay documentation
Add screenshots
Signed-off-by: Smriti S <smriti.satyanarayana@harness.io>

### CODE REVIEW

File: docs/chaos-engineering/configure-chaos-experiments/gameday/run-gameday.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+### Step 5: Save the GameDay
+After adding the experiments to the GameDay, click **Save**.
+![](./static/run-gameday/9-save-experiment.png)
+
+### Step 6: Schedule or run the GameDay
+1. Now that you have the experiments in the GameDay, you are all set to run them. 
+![](./static/run-gameday/10-gameday-created.png)
+
+2. Click **Start GameDay**. This prepares your GameDay to run the experiments within the GameDay.
```
Comment (Saranya-jena) [inline docs/chaos-engineering/configure-chaos-experiments/gameday/run-gameday.md:89]: On clicking `start gameday` it actually *creates* the experiment. And after going inside a particular gameday run, we can run the experiments. 
Maybe we can mention the experiment creation part?

File: docs/chaos-engineering/configure-chaos-experiments/gameday/run-gameday.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+5. Click **Run Experiment**. This begins the execution of the experiment. 
+![](./static/run-gameday/14-run-experiment.png)
+
+
+### Step 7: Record the conclusion and action items
+1. You can add a summary of the experiments in the GameDay by clicking **Add/View Notes**.  
+![](./static/run-gameday/15-summary-at-exp-level.png)
+
+2. You can mark an experiment run as complete by clicking **Mark Run as Complete**. This way, you will not be able to run the experiment again. This completed run serves as a checkpoint within the GameDay that helps keep track of how the application fared during a specific situation.
```
Comment (Saranya-jena) [inline docs/chaos-engineering/configure-chaos-experiments/gameday/run-gameday.md:106]: - You can mark an experiment run as complete -> You can mark gameday run as complete
We can rephrase this entire paragraph. We actually mark a gameday run as complete, once marked we won't be able to rerun the experiments inside that gameday run. We will have to start a new gameday run.

**Review by Saranya-jena — CHANGES_REQUESTED** [review]

File: docs/chaos-engineering/configure-chaos-experiments/gameday/run-gameday.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+### Step 3: Specify the details of the GameDay
+Add details such as **GameDay name**, **Objectives**, and **Description** (optional). Click **Next-> Select Chaos Experiments**.
+![](./static/run-gameday/2-create-new-gameday.png)
+
+### Step 4: Add experiments to the GameDay
+1. After creating a GameDay, add experiments to the GameDay by clicking **New Chaos Experiment**.
+![](./static/run-gameday/3-add-experiments-to-gameday.png)
+
+2. You can add experiments by selecting the ones available in the chaos hub.
```
[the remark this answers is not in the document]
Comment (SmritiSatya) [reply docs/chaos-engineering/configure-chaos-experiments/gameday/run-gameday.md:65]: I think we are changing chaoshub to chaos hub.

[the remark this answers is not in the document]
Comment (Saranya-jena) [reply docs/chaos-engineering/configure-chaos-experiments/gameday/run-gameday.md:68]: okay, I wasn't aware of that. Resolving all the similar comments

File: docs/chaos-engineering/configure-chaos-experiments/gameday/run-gameday.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+7. Record the conclusion and action items
+
+:::info
+Once you create a GameDay in HCE, you can run it as many times as you wish. HCE saves the information about every run, which includes the date, summary, any notes you add, and so on.
+:::
+
+## Prerequisites
+You can create a GameDay provided you have the following prerequisites.
+1. HCE account
+2. Access to a cluster
```
Comment (deepakbolangady) [inline docs/chaos-engineering/configure-chaos-experiments/gameday/run-gameday.md:42]: Is this a K8s cluster? If yes, can we specify that? Which brings me to my next question: are we supporting GameDays only with K8s clusters today?

File: docs/chaos-engineering/configure-chaos-experiments/gameday/run-gameday.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+
+## Creating a GameDay
+You can create and run a GameDay with the following steps.
+
+### Step 1: Plan your GameDay
```
Comment (deepakbolangady) [inline docs/chaos-engineering/configure-chaos-experiments/gameday/run-gameday.md:47]: Per our style guide, begin with the most meaningful words. Do not include “Step”, “Review”, “Option” at the beginning of headings.

File: docs/chaos-engineering/configure-chaos-experiments/gameday/run-gameday.md [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+
+This is one of the critical questions that needs to be addressed before proceeding to run a Gameday. The GameDay should address questions such as:
+1. Which services will you test?
+2. What is the goal of the GameDay? 
+3. What will you verify or determine by the end of the GameDay?
```
Comment (deepakbolangady) [inline docs/chaos-engineering/configure-chaos-experiments/gameday/run-gameday.md:52]:
```suggestion
Address the following questions before proceeding to run a Gameday:
1. Which services should I test?
2. What is the goal of the GameDay? 
3. What should I verify or determine by the end of the GameDay?
```

File: docs/chaos-engineering/configure-chaos-experiments/gameday/run-gameday.md [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+### Step 2: Create a GameDay
+To create a GameDay, click **+New GameDay**. 
+![](./static/run-gameday/1-landing-page.png)
+
+### Step 3: Specify the details of the GameDay
+Add details such as **GameDay name**, **Objectives**, and **Description** (optional). Click **Next-> Select Chaos Experiments**.
+![](./static/run-gameday/2-create-new-gameday.png)
+
+### Step 4: Add experiments to the GameDay
+1. After creating a GameDay, add experiments to the GameDay by clicking **New Chaos Experiment**.
```
Comment (deepakbolangady) [inline docs/chaos-engineering/configure-chaos-experiments/gameday/run-gameday.md:65]: About "After creating a GameDay": have we created the GameDay at this point? I thought adding experiments was part of creating the GameDay (per the seven-step process outlined earlier).

File: docs/chaos-engineering/configure-chaos-experiments/gameday/run-gameday.md [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+![](./static/run-gameday/3-add-experiments-to-gameday.png)
+
+2. You can add experiments by selecting the ones available in the chaos hub.  
+![](./static/run-gameday/4-list-experiments-from-chaoshub.png)
+
+3. Select a chaos infrastructure. Click **Add experiments to the GameDay**.
+![](./static/run-gameday/6-add-experiments-to-gameday.png)
+
+:::info
+You can add up to 20 experiments to every GameDay, and every chaos experiment can be executed on a different chaos infrastructure!
```
Comment (deepakbolangady) [inline docs/chaos-engineering/configure-chaos-experiments/gameday/run-gameday.md:75]:
```suggestion
You can add up to 20 experiments to every GameDay, and every chaos experiment can be executed on a different chaos infrastructure.
```

File: docs/chaos-engineering/configure-chaos-experiments/gameday/run-gameday.md [hunk 9]
```diff
@@ -0,0 +1,114 @@
...
+:::
+
+4. This will lead you to a page that lists the experiments you selected for the current GameDay. You can add or delete experiments from this page.
+![](./static/run-gameday/7-add-multiple-experiments.png)
+
+5. To view the pipeline of an experiment, you can select one by clicking it. This displays a preview of the experiment.
```
Comment (deepakbolangady) [inline docs/chaos-engineering/configure-chaos-experiments/gameday/run-gameday.md:81]:
```suggestion
5. To view a preview of the experiment's pipeline, select the experiment.
```
(Is that accurate?)

**Review by deepakbolangady — CHANGES_REQUESTED** [review]

File: docs/chaos-engineering/configure-chaos-experiments/gameday/run-gameday.md [hunk 10]
```diff
@@ -0,0 +1,114 @@
...
+It typically involves three steps:
+
+1. Run chaos experiments on your application
+2. Observe the impact of the failure
+3. Discuss the technical outcomes
+
+:::tip
+GameDays help decide the type of failure the system would undergo. Hence, it is strongly recommended to begin with easy use cases where the blast radius is minimal, such as breaking one container, degrading one instance, making one availability zone unavailable, and so on. Later on, you can delve into more complex failures, such as failing an entire service or affecting a large percentage of requests, and so on.
+
+At every step in implementing a GameDay, you will either find a glitch in the system that is addressed or gain confidence in your application.
```
[the remark this answers is not in the document]
Comment (SmritiSatya) [reply docs/chaos-engineering/configure-chaos-experiments/gameday/run-gameday.md:21]: Yes, @deepakbolangady

File: docs/chaos-engineering/configure-chaos-experiments/gameday/run-gameday.md [hunk 11]
```diff
@@ -0,0 +1,114 @@
...
+title: Run a GameDay
+sidebar_position: 1
+---
+This section describes what a GameDay is, how a typical GameDay is run, and the outcomes of it. 
+
+## Introduction
+
+GameDay is a methodology to execute chaos experiments in your application during a specific time period. It is a template that helps you schedule and execute one or more chaos experiments within your application. GameDays help apply a fatal scenario to your application in a safe environment, thereby determining the resilience of the application. 
+
+Running a GameDay verifies the system at scale, and in production.
```
[the remark this answers is not in the document]
Comment (SmritiSatya) [reply docs/chaos-engineering/configure-chaos-experiments/gameday/run-gameday.md:11]: I somehow think production environment isn't safe, so I removed that sentence. Safe environment is something that is not in production I believe.

File: docs/chaos-engineering/configure-chaos-experiments/gameday/run-gameday.md (hunk 4 above)
Comment (SmritiSatya) [reply docs/chaos-engineering/configure-chaos-experiments/gameday/run-gameday.md:42]: Any experiment would run on cluster, but the type depends on the platform- AWS, Kubernetes, etc. Hence I did not specify K8s.

File: docs/chaos-engineering/configure-chaos-experiments/gameday/run-gameday.md (hunk 7 above)
Comment (SmritiSatya) [reply docs/chaos-engineering/configure-chaos-experiments/gameday/run-gameday.md:65]: By 'creating a GameDay', we are creating a template. Adding experiments is another step. I will rephrase it.

File: docs/chaos-engineering/configure-chaos-experiments/gameday/run-gameday.md (hunk 9 above)
Comment (SmritiSatya) [reply docs/chaos-engineering/configure-chaos-experiments/gameday/run-gameday.md:81]: Can we say "5. Select the experiment to see a preview of the experiment's pipeline."?

**Review by Jonsy13 — APPROVED** [review]

**Pull request merged**